### DISCUSSION TITLE
My friend removed me from a github organisation I had my projects on

### DISCUSSION THREAD

**Discussion by lbirkert** [community/community#59674] [Repositories] [answered]
### Select Topic Area

Question

### Body

Recently my friend decided to block me and remove me from our github organisation where *I was the only contributor*. Can I somehow remote remove these repositories/commits from this github organisation and move them to my account?

Thanks for your help.

**Comment by seo-asif** [comment, marked as answer]
If your friend has blocked you and removed you from the GitHub organization where you were the sole contributor, retrieving the repositories and commits can be challenging. However, you can try the following steps:

1. Contact your friend: Reach out to your friend and explain the situation. It's possible that the removal was unintentional or a result of a misunderstanding. Communicate with them and request access to the repositories or discuss the possibility of transferring ownership.

2. Contact GitHub Support: If you are unable to resolve the issue with your friend, you can reach out to GitHub Support for assistance. Explain the situation to them, provide any relevant details, and request their help in regaining access to the repositories.

3. Provide evidence of ownership: If you were the sole contributor and can prove your ownership of the repositories, gather any evidence you have to support your claim. This may include commit history, documentation, or any other relevant information that demonstrates your involvement and contribution to the repositories.

4. Explore legal options: In extreme cases where access to your repositories is denied and other avenues have been exhausted, you may need to consult legal counsel to explore potential legal actions to regain access to your repositories. This step should only be considered if the repositories hold significant value or if you believe your rights have been violated.
